<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label_avant">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>480</width>
     <height>360</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>480</width>
     <height>360</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>480</width>
     <height>360</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_apres">
   <property name="geometry">
    <rect>
     <x>496</x>
     <y>8</y>
     <width>480</width>
     <height>360</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>480</width>
     <height>360</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>480</width>
     <height>360</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bouton_pause">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>400</y>
     <width>106</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>pause</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_erreur">
   <property name="geometry">
    <rect>
     <x>496</x>
     <y>372</y>
     <width>477</width>
     <height>84</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
